Utils/Elasticsearch/mapping/prodsys_rucio.mapping

```diff
@@ -177,6 +173,30 @@
           }
         }
       }
+    },
+    {
+        "output_dataset": {
+            "_parent": {
```

Comment:
Two moments:
* broken structure: excessive `{...}` around `"output_datasets"`;
* broken style: indentation changed from 2 spaces to 4.

The structure must look this way:
```
{
  "template": "prodsys_rucio",
  "settings": {
    "number_of_shards": 4
  },
  "mappings" : {
    "task" : {
      "properties" : {
        "campaign" : {
          ...
        }
      }
    },
    "output_dataset": {
      ...
    }
  }
}